README.md

```diff
@@ -0,0 +1,21 @@
+## cloud-provider-aws
+Kubernetes on AWS is great. A properly configured cluster can handle the creation and management of EBS volumes, ELBs for services, Security Groups, and even networking interfaces.
+
+## AWS Labels
```

Comment:
Based on my reading of https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/tags.go, this describes the 'legacy' behavior.  the current standard seems to be setting 'kubernetes.io/cluster/ClusterID' to either 'shared' or 'owned'

Reply:
@sroylance @nckturner  sorry, i couldn't parse. Can you please elaborate a bit more, so that i can make appropriate changes.